xrmocap/model/registrant/smplifyx.py

```diff
@@ -36,11 +40,24 @@ def __set_keypoint_indexes__(self) -> None:
         self.body_keypoint_idxs = get_keypoint_idxs_by_part(
             'body', convention=convention)
 
+        self.shoulder_keypoint_idxs = get_keypoint_idxs_by_part(
+            'shoulder', convention=convention)
+
+        self.hip_keypoint_idxs = get_keypoint_idxs_by_part(
+            'hip', convention=convention)
+
+        self.foot_keypoint_idxs = get_keypoint_idxs_by_part(
+            'foot', convention=convention)
+
     def get_keypoint_weight(self,
                             use_shoulder_hip_only: bool = False,
                             body_weight: float = 1.0,
                             hand_weight: float = 1.0,
-                            face_weight: float = 1.0) -> torch.Tensor:
+                            face_weight: float = 1.0,
+                            shoulder_weight: float = 1.0,
+                            hip_weight: float = 1.0,
+                            foot_weight: float = 1.0,
+                            **kwargs) -> torch.Tensor:
         """Get per keypoint weight.
 
         Args:
```

Comment:
docstring not updated

Reply:
Fixed in [f33d0ca](https://github.com/openxrlab/xrmocap/pull/52/commits/f33d0caed97adb0a3c2afe553b5f9f34503f900f).